components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx

```diff
@@ -282,8 +282,8 @@ export default class WorkerSpecific extends React.Component {
                     }, 1000);
                 }
             }).catch((error) => {
-                that.setState({open: false});
-                that.showError("Error while changing statistics configuration!!");
+            that.setState({open: false});
+            that.showError("Error while changing statistics configuration!!");
```

Comment:
Shall we change this error message to be more informative & user-friendly. e.g. "An error occurred when changing statistic dashboard configurations"